### ISSUE TITLE
EPIC: Prepare the Project Pages page for Homepage Launch

### ISSUE THREAD

**Issue by priyanka02art** [hackforla/website#7679]
### Overview
We need to track existing issues and make new issues to deliver Project Pages page complete enough for the Homepage launch

#### Details
We need to track and support the final wrap up items for the homepage launch, so that it can be completed.

### Action Items
- [x] List all the issues related to the label
- [x] Identify what open issues have to be resolved before launch, by classifying the issues into the following categories using MoSCoW method in a comment.
   - Must Have
   - Should Have
   - Could Have
   - Won't Have this time
- [x] Add the following details under each issue
   ```
   - why: [ADD WHY ITS IMPORTANT TO TO THE LAUNCH]
   - status: [ADD STATUS e.g., Icebox, Emergent Requests, New Issue Approval, ERs and epics that are ready to be turned into issues, Prioritized Backlog, In Progress (actively working), Questions / In Review]
      - draft: [true OR false]
      - responsible: [ADD GITHUB HANDLES WITHOUT @]
   - role: [ADD ROLE e.g. data scientist, infrastructure, design, design lead, user research, Research Lead, dev leads, data analyst, BA, hfla leadership, back end/devOps, writing, front end, merge team, technical writer]
   ```
- [x] Once you receive sign-off from Staff PM add to `## Relevance of the issues to the Homepage launch`
- [x] Review comments with `Notes carried forward from epic` if it exists, and make sure all previous items are addressed in the top of the issue.
   - If it specifies new issues to make then add explicit action items for making those issue in a new comment on this issue
      - get sign-off on those new issues to make by Staff PM 
- [x] Identify items that need issues opened by looking at the page and features and seeing what is not already addressed by the existing issues or new issues to make list.
- [x] Make the issues specified
- [x] Link them as sub issues to this issue
- [x] Classify them using MoSCoW and add the details (why, status, draft, responsible, role)
- [ ] Get Sign-off and prioritization of the issues by Staff PM
- [ ] If this page has content that must be checked for accuracy before homepage launch, add the detail to the epic for tracking that (resource 1.05)

### Resources/Instructions
- Parent Epic: https://github.com/hackforla/website/issues/2481

#### Used for creating this issue
- 1.01 Label name: P-Feature: Project Info and Page
    - 1.01.01 Issue Tab: [P-Feature: Project Info and Page](https://github.com/hackforla/website/issues?q=is%3Aopen+label%3A%22P-Feature%3A+Project+Info+and+Page%22)
    - 1.01.02 Project Board: [P-Feature: Project Info and Page](https://github.com/orgs/hackforla/projects/86/views/1?filterQuery=+label%3A%22P-Feature%3A+Project+Info+and+Page%22)
- 1.02 URL of page on Hack for LA website: A project's detail page (e.g. https://www.hackforla.org/projects/100-automations)
- 1.03 URL of link to code search: [repo:hackforla/website NOT language:SVG NOT path:/^_data/internal/credits// Project Info and Page](https://github.com/search?q=repo%3Ahackforla%2Fwebsite+NOT+language%3ASVG+NOT+path%3A%2F%5E_data%5C%2Finternal%5C%2Fcredits%5C%2F%2F+ProjectInfoAndPage&type=code)
- 1.04 URL of WIKI page: https://github.com/search?q=repo%3Ahackforla%2Fwebsite+Project+Info+and+Page&type=wikis
   - 1.04.01 URL of future edits to wiki pages: https://github.com/hackforla/website/wiki/How-to-Contribute-to-the-Wiki (check this page to see if there are any other new edits in the queue.)
- 1.05 Issue for adding items to a final checklist: #8417

#### Created by this issue
- 2.01 [EXAMPLE would be a link to an issue related to delivering this page]
---
### Must Have
TBD

### Should Have
TBD

### Could Have
TBD

---

last updated: 2026-04-21
Highest number found: 8560

### Must Have
- [ ] #7679
   - why: This issue
   - status: In progress (actively working)
      - draft: False
      - responsible: nadiaspn0503 
   - role: product

 - [ ] #8560
   - why: Need to have Project Pages page with correct information.
   - status: New Issue Approval
      - draft: True
      - responsible: None 
   - role: back end/devOps and front end

 - [ ] #8543
   - why: We need to create a template to create good first issues to populate the github-handle variable with the correct handle for each team member (excluding leadership team members).
   - status: New Issue Approval
      - draft: True
      - responsible: None 
   - role: back end/devOps and front end

 - [ ] #8541
   - why: Need to have Project Pages page with correct information.
   - status: Ready for Prioritization 
      - draft: False
      - responsible: None 
   - role: back end/devOps and front end

 - [ ] #8523
   - why: Need to have Project Pages page with correct information.
   - status: Ready for Prioritization 
      - draft: False
      - responsible: None 
   - role: back end/devOps and front end

 - [ ] #8522
   - why: Need to have Project Pages page with correct information.
   - status: Ready for Prioritization 
      - draft: False
      - responsible: None 
   - role: back end/devOps and front end

 - [ ] #8521
   - why: Need to have Project Pages page with correct information.
   - status: Ready for Prioritization 
      - draft: False
      - responsible: None 
   - role: back end/devOps and front end

 - [ ] #8520
   - why: Need to have Project Pages page with correct information.
   - status: Ready for Prioritization 
      - draft: False
      - responsible: None 
   - role: back end/devOps and front end

 - [ ] #8450
   - why: Need to have Project Pages page with correct information.
   - status: Prioritized backlog
      - draft: False
      - responsible: None 
   - role: back end/devOps and front end

 - [ ] #8407
   - why: Need to ensure that text used across the projects follows consistent tone, terminology, and style conventions
   - status: New Issue Approval
      - draft: True
      - responsible: None 
   - role: product and writing

 - [ ] #8361
   - why: Need to have Project Pages page with correct information.
   - status: Prioritized backlog
      - draft: False
      - responsible: none 
   - role: back end/devOps and front end

 - [ ] #8348
   - why: Need to put the most recent One Sheet on the website.
   - status: New Issue Approval
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

 - [ ] #8346
   - why: Need to have Project Pages page with correct information.
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

 - [ ] #8332
   - why: Need to have Project Pages page with correct information.
   - status: Prioritized backlog
      - draft: False
      - responsible: none
   - role: back end/devOps and front end

- [ ] #8287
   - why: Need to have Project Pages page with correct information.
   - status: In progress (actively working)
      - draft: False
      - responsible: Tomlu60220244 
   - role: back end/devOps and front end

- [ ] #8054
   - why: Need to have Project Pages page with correct information.
   - status: New Issue Approval - LM 2025-09-17
      - draft: True
      - responsible: JessicaLucindaCheng 
   - role: dev leads, merge team, and product

- [ ] #6138
   - why: To change the HfLA website code to read from as many repositories as specified per project
   - status: In progress (actively working)
      - draft: False
      - responsible: JessicaLucindaCheng 
   - role: back end/devOps and front end

- [ ] #5127
   - why: To update the languages for Access the Data so that visitors to the website can find accurate and up-to-date information.
   - status: Ice box
      - draft: False
      - responsible: None 
   - role: back end/devOps and front end

- [ ] #4153
   - why: Move, refactor and add SDG info and images for the rollout plan.
   - status: In progress (actively working)
      - draft: False
      - responsible: JessicaLucindaCheng
   - role: Dev leads

### Should Have
- [ ] #6277
   - why: To reduce duplicate data
   - status: In progress (actively working)
      - draft: False
      - responsible: JessicaLucindaCheng 
   - role: back end/devOps

- [ ] #6144
   - why: Need to change the code for displaying contribution data for each person in a project that has 2 or more repositories (repos). This is used for the All-Time Contributors section of each project's detailed page.
   - status: New Issue Approval
      - draft: True
      - responsible: JessicaLucindaCheng 
   - role: back end/devOps

- [ ] #5630
   - why: Makes it easier to convert stakeholder requests into actionable developer issues.
   - status: New Issue Approval
      - draft: True
      - responsible: JessicaLucindaCheng 
   - role: back end/devOps and front end

### Could Have
<details><summary>List of all could have issues</summary>

- #8428
   - why: Create issues to populate the github-handle variable with the correct handle for each member of the leadership team
   - status: In progress (actively working)
      - draft: False
      - responsible: kdaca19xx and xnealcarson 
   - role: back end/devOps and front end

- #7801
   - why: Create a single variable to hold a GitHub handle
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

- #7799
   - why: Create a single variable to hold a GitHub handle
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

- #7795
   - why: Create a single variable to hold a GitHub handle
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

- #7794
   - why: Create a single variable to hold a GitHub handle
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

- #7781
   - why: Create a single variable to hold a GitHub handle
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

- #7765
   - why: Create a single variable to hold a GitHub handle
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

- #7764
   - why: Create a single variable to hold a GitHub handle
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

- #7763
   - why: Create a single variable to hold a GitHub handle
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

- #7435
   - why: Create a single variable to hold a GitHub handle
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

- #7434
   - why: Create a single variable to hold a GitHub handle
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

- #7432
   - why: Create a single variable to hold a GitHub handle
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

- #7431
   - why: Create a single variable to hold a GitHub handle
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

- #7430
   - why: Create a single variable to hold a GitHub handle
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

- #7402
   - why: Create a single variable to hold a GitHub handle
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

- #7401
   - why: Create a single variable to hold a GitHub handle
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

- #7384
   - why: Create a single variable to hold a GitHub handle
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

- #7102
   - why: Create a single variable to hold a GitHub handle
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

- #6820
   - why: Dates needed for the future but are not currently needed.
   - status: ERs and epics that are ready to be turned into issues
      - draft: False
      - responsible: None
   - role: back end/devOps

- #6819
   - why: To reduce scattered information and solve an emerging problem
   - status: Ice box
      - draft: True
      - responsible: ExperimentsInHonesty 
   - role: back end/devOps
   
- #6321
   - why: To make it easy for future team members to edit.
   - status: Ice box
      - draft: False
      - responsible: None
   - role: front end
  
- #6139
   - why: To update the languages for Access the Data for website users to find accurate and up-to-date information
   - status: In progress (actively working)
      - draft: False
      - responsible: JessicaLucindaCheng 
   - role: back end/devOps and front end

- #6017
   - why: Ensures meetings display correctly on Project pages
   - status: New Issue Approval
      - draft: True
      - responsible: None
   - role: back end/devOps

- #5441
   - why: Create a single variable to hold a GitHub handle
   - status: In progress (actively working)
      - draft: False
      - responsible: daras-cu 
   - role: back end/devOps and front end

- #5440
   - why: Create a single variable to hold a GitHub handle
   - status: Ice box
      - draft: False
      - responsible: None
   - role: back end/devOps

- #4262
   - why: To identify and update the technologies and tools requirement for all the projects.
   - status: In progress (actively working)
      - draft: False
      - responsible: kirandhamotharan 
   - role: product

- #2037
   - why: To add information to each project md file to be displayed on the new Impact format.
   - status: New Issue Approval
      - draft: True
      - responsible: None
   - role: product

- #312
   - why: Provide collateral for the HackforLA website
   - status: New Issue Approval
      - draft: False
      - responsible: None
   - role: product

</details> 

### Won't have this time
<details><summary>List of all won't have issues</summary>

- #7726
- #4342
- #3526

</details> 

#### Templates
```
- [ ] #
   - why: 
   - status: 
      - draft: 
      - responsible: 
   - role:  
```
```
### Issues list associated with this epic
last updated: 
Highest number found:
```

<details><summary>List of all issue by number</summary>

- #8560
- #8543
- #8541
- #8523
- #8522
- #8521
- #8520
- #8450
- #8428
- #8407
- #8361
- #8348
- #8346
- #8332 
- #8287 
- #8054
- #7801
- #7799
- #7795
- #7794
- #7781
- #7765
- #7764
- #7763
- #7726
- #7679
- #7435
- #7434
- #7432
- #7431
- #7430
- #7402
- #7401
- #7384
- #7102
- #6820
- #6819
- #6321
- #6277
- #6144
- #6139
- #6138
- #6017
- #5630 
- #5441
- #5440
- #5127
- #4342
- #4262
- #4153
- #3526
- #2037
- #312

</details>

**Comment by priyanka02art** [comment]
- Access the Data
   - [x] https://github.com/hackforla/website/issues/5127
   - [x] https://github.com/hackforla/website/issues/6138
   - [x] https://github.com/hackforla/website/issues/6144
   - [x] https://github.com/hackforla/website/issues/5337
- Expunge Assist
   - [x] https://github.com/hackforla/website/issues/6834
   - [x] https://github.com/hackforla/website/issues/6866
   - [x] https://github.com/hackforla/website/issues/6579
- New Project
   - [ ] https://github.com/hackforla/website/issues/4342
- All project pages
   - [ ] https://github.com/hackforla/website/issues/6838
   - [ ] https://github.com/hackforla/website/issues/6930
   - [ ] https://github.com/hackforla/website/issues/6129
   - [ ] https://github.com/hackforla/website/issues/6930
   - [ ] https://github.com/hackforla/website/issues/6920
   - [ ] https://github.com/hackforla/website/issues/6655

**Comment by priyanka02art** [comment]
# List of all issues
- [ ] #7990 
- [ ] #6144 
- [ ] #6139 
- [ ] #5127 
- [ ] #4342 
- [ ] #4262 
- [ ] #4153 
- [ ] #2037 
- [ ] #312 

## Relevance of the issues to the Homepage launch
### Must Have
- [ ] All issue with "Updated Project Profile" need to completed before the launch 
   - why: There will be a lot of attention outdated information about peoples role. 
   - status: 
      - draft
      - responsible: 
   - role:
- [ ] #7679   
   - why: It is the epic for this page (this issue)
   - status:  
      - draft:  
      - responsible:  
   - role:   
- [ ] #5127
   - why: To update the languages for Access the Data so that visitors to the website can find accurate and up-to-date information.
   - status: Ice box
      - draft: False
      - responsible: none
   - role: front end
- [ ] #4153
   - why: Move, refactor and add SDG info and images for the rollout plan.
   - status: In progress (actively working)
      - draft: False
      - responsible: JessicaLucindaCheng  
   - role: dev leads

### Should Have
- [ ] #7990
   - why: Allowing user to re-center the map on click, instead of displaying tooltip over the disabled button.
   - status:  new issue approval
      - draft:  False
      - responsible:  none
   - role: front end
- [ ] #6144
   - why: Need to change the code for displaying contribution data for each person in a project that has 2 or more repositories (repos). This is used for the All-Time Contributors section of each project's detailed page.
   - status: new issue approval 
      - draft: True
      - responsible:  JessicaLucindaCheng 
   - role: back end/devOps
### Could Have
- [ ] #6139
   - why: To update the languages for Access the Data for website users to find accurate and up-to-date information.
   - status:  In progress(actively working)
      - draft:  False 
      - responsible:  JessicaLucindaCheng 
   - role: back end/devOps
- [ ] #4262
   - why: To identify and update the technologies and tools requirement for all the projects.
   - status: new issue approval
      - draft: False
      - responsible: none
   - role: product
 - [ ] #2037
   - why: To add information to each project md file to be displayed on the new Impact format.
   - status: new issue approval
      - draft: True
      - responsible: none
   - role: product
  - [ ] #312
    - why: Provide collateral for the HackforLA website
    - status: new issue approval
      - draft: False
      - responsible: none
    - role: product
   

### Won't have this time
- [ ] #4342
- [ ] # 
   - why: 
   - status:  
      - draft:  
      - responsible:  
   - role:
#### Templates
```
- [ ] # 
   - why: 
   - status: 
      - draft
      - responsible: 
   - role: 
```
```
### Issues list associated with this epic
last updated: 
Highest number found:
```

**Comment by priyanka02art** [comment]
# List of all issues
- [ ] #7990 
- [ ] #6144 
- [ ] #6139 
- [ ] #5127 
- [ ] #4342 
- [ ] #4262 
- [ ] #4153 
- [ ] #2037 
- [ ] #312

**Comment by ExperimentsInHonesty** [comment]
@sneha-kataria Please provide update

**Comment by nadiaspn0503** [comment]
### Issues list associated with this epic
last updated: 2025-09-16
Highest number found: 8303
- #8303
- #8287 
- #8282 
- #8278
- #8117
- #8111
- #8054
- #7802
- #7801
- #7800
- #7799
- #7795
- #7794
- #7781
- #7765
- #7764
- #7763
- #7726
- #7435
- #7434
- #7432
- #7431
- #7430
- #7402
- #7401
- #7384
- #7382
- #7357
- #7354
- #7353
- #7352
- #7337
- #7334
- #7333
- #7332
- #7289
- #7102
- #6820
- #6819
- #6412
- #6321
- #6277
- #6017
- #5630 
- #5441
- #5440
- #3526

---
### Must have
- #8303
   - why: Need to have Project Pages page with correct information.
   - status: In Progress / Actively Working
      - draft: False
      - responsible: rogerioduenas 
   - role: back end/devOps and front end

- #8287
   - why: Need to have Project Pages page with correct information.
   - status: In Progress / Actively Working
      - draft: False
      - responsible: Jeffreyp1 
   - role: back end/devOps and front end

- #8282
   - why: Need to have Project Pages page with correct information.
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

- #8278
   - why: Need to have Project Pages page with correct information.
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

- #8117
   - why: Need to have Project Pages page with correct information.
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

- #8111
   - why: Need to have Project Pages page with correct information.
   - status: Questions
      - draft: False
      - responsible: Was assigned to MohamedTwahir who experienced a technical problem added ready for merge team label 
   - role: back end/devOps and front end

- #8054
   - why: Need to have Project Pages page with correct information.
   - status: New Issue Approval - LM 2025-09-17
      - draft: True
      - responsible: JessicaLucindaCheng 
   - role: dev leads, merge team, and product

- #6412
   - why: Need to have Project Pages page with correct information.
   - status: ERs and epics that are ready to be turned into issues
      - draft: False
      - responsible: None (Added to https://github.com/hackforla/website/issues/7859#issuecomment-3330185206)
   - role: back end/devOps
  
### Should have
- #6277
   - why: To reduce duplicate data
   - status: In progress (actively working)
      - draft: False
      - responsible: JessicaLucindaCheng 
   - role: back end/devOps

- #5630
   - why: Makes it easier to convert stakeholder requests into actionable developer issues.
   - status: New Issue Approval
      - draft: True
      - responsible: JessicaLucindaCheng 
   - role: back end/devOps and front end

### Could have
- #7802
   - why: Create a single variable to hold a GitHub handle
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

- #7801
   - why: Create a single variable to hold a GitHub handle
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

- #7800
   - why: Create a single variable to hold a GitHub handle
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

- #7799
   - why: Create a single variable to hold a GitHub handle
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

- #7795
   - why: Create a single variable to hold a GitHub handle
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

- #7794
   - why: Create a single variable to hold a GitHub handle
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

- #7781
   - why: Create a single variable to hold a GitHub handle
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

- #7765
   - why: Create a single variable to hold a GitHub handle
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

- #7764
   - why: Create a single variable to hold a GitHub handle
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

- #7763
   - why: Create a single variable to hold a GitHub handle
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

- #7435
   - why: Create a single variable to hold a GitHub handle
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

- #7434
   - why: Create a single variable to hold a GitHub handle
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

- #7432
   - why: Create a single variable to hold a GitHub handle
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

- #7431
   - why: Create a single variable to hold a GitHub handle
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

- #7430
   - why: Create a single variable to hold a GitHub handle
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

- #7402
   - why: Create a single variable to hold a GitHub handle
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

- #7401
   - why: Create a single variable to hold a GitHub handle
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

- #7384
   - why: Create a single variable to hold a GitHub handle
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

- #7382
   - why: Create a single variable to hold a GitHub handle
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

- #7357
   - why: Create a single variable to hold a GitHub handle
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

- #7354
   - why: Create a single variable to hold a GitHub handle
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

- #7353
   - why: Create a single variable to hold a GitHub handle
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

- #7352
   - why: Create a single variable to hold a GitHub handle
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

- #7337
   - why: Create a single variable to hold a GitHub handle
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

- #7334
   - why: Create a single variable to hold a GitHub handle
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

- #7333
   - why: Create a single variable to hold a GitHub handle
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

- #7332
   - why: Create a single variable to hold a GitHub handle
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

- #7289
   - why: Create a single variable to hold a GitHub handle
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

- #7102
   - why: Create a single variable to hold a GitHub handle
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

- #6321
   - why: To make it easy for future team members to edit.
   - status: Ice box
      - draft: False
      - responsible: None
   - role: front end

- #6820
   - why: Dates needed for the future but are not currently needed.
   - status: ERs and epics that are ready to be turned into issues
      - draft: False
      - responsible: None
   - role: back end/devOps

- #6819
   - why: To reduce scattered information and solve an emerging problem
   - status: Emergent Requests
      - draft: True
      - responsible: @ExperimentsInHonesty 
   - role: back end/devOps

- #6017
   - why: Ensures meetings display correctly on Project pages
   - status: New Issue Approval
      - draft: True
      - responsible: None
   - role: back end/devOps

- #5441
   - why: Create a single variable to hold a GitHub handle
   - status: New Issue Approval
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

- #5440
   - why: Create a single variable to hold a GitHub handle
   - status: Ice box
      - draft: False
      - responsible: None
   - role: back end/devOps

### Won't have
- #7726
   - why: Need to have Project Pages page with correct information.
   - status: Ice Box
      - draft: True
      - responsible: None
   - role: back end/devOps and front end

- #3526
   - why: Ensures alignment with the Design System
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: design

**Comment by nadiaspn0503** [comment]
### Issues list associated with this epic
last updated: 2025-10-1
Highest number found: 8350
- #8350
- #8348
- #8347
- #8346
- #8344
- #8337
- #8334
- #8333
- #8332
- #8330 
- #8329
- #8322
- #8318
- #8314

--- 
### Must have
 - #8350
   - why: Need to have Project Pages page with correct information.
   - status: New Issue Approval
      - draft: False
      - responsible: kdaca19xx 
   - role: back end/devOps and front end

 - #8348
   - why: Need to put the most recent One Sheet on the website.
   - status: New Issue Approval
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

 - #8347
   - why: Need to have Project Pages page with correct information.
   - status: New Issue Approval
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

 - #8346
   - why: Need to have Project Pages page with correct information.
   - status: New Issue Approval
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

 - #8344
   - why: Need to list only active projects
   - status: New Issue Approval
      - draft: False
      - responsible: None
   - role: back end/devOps

 - #8337
   - why: Need to have Project Pages page with correct information.
   - status: New Issue Approval
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

 - #8334
   - why: Need to have Project Pages page with correct information.
   - status: New Issue Approval
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

 - #8333
   - why: Need to have Project Pages page with correct information.
   - status: New Issue Approval
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

 - #8332
   - why: Need to have Project Pages page with correct information.
   - status: New Issue Approval
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

 - #8330
   - why: Need to have Project Pages page with correct information.
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

 - #8329
   - why: Need to have Project Pages page with correct information.
   - status: New Issue Approval
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

 - #8322
   - why: Need to have Project Pages page with correct information.
   - status: New Issue Approval
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

 - #8318
   - why: Need to have Project Pages page with correct information.
   - status: New Issue Approval
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

 - #8314
   - why: Need to have Project Pages page with correct information.
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

### Should have

### Could have

### Won't have

**Comment by nadiaspn0503** [comment]
### BOP
- @ExperimentsInHonesty Need to review this issue https://github.com/hackforla/website/issues/7726

### Access the Data
- @ExperimentsInHonesty Need to review this issue https://github.com/hackforla/website/issues/5337

**Comment by nadiaspn0503** [comment]
We reviewed all of the must haves except for the last two. Next week we will finish reviewing the must haves and the should haves and could haves.

**Comment by nadiaspn0503** [comment]
Check all of the adding GitHub handles. Left off at 7402. Make sure they are all in the Prioritized Backlog.

**Comment by nadiaspn0503** [comment]
2026-08-18
Using this search 
[state:open label:"P-Feature: Project Info and Page" is:issue ](https://github.com/hackforla/website/issues?q=state%3Aopen%20label%3A%22P-Feature%3A%20Project%20Info%20and%20Page%22%20is%3Aissue)

This is the highest issue number found 8737 it is addressed at the top of this issue.

**Comment by kimberlyvaldez07** [comment]
11/5 - make a new issue for this and it will be discussed in the next meeting.

Title:
Review and align language across documentation and interface elements
Overview:
We need to ensure that text used across the project follows consistent tone, terminology, and style conventions. This review will identify any inconsistencies in phrasing and recommend updates to improve clarity and alignment with our communication standards.
